docs/user/troubleshooting/03-00-istio-basic-diagnostics.md

```diff
@@ -0,0 +1,26 @@
+# Istio Basic Diagnostics
+
+If you are experiencing issues with the cluster, you can follow the steps below to troubleshoot the problem before you contact us. This guide is designed to help you identify the root cause of the problem which might not necessarily be related to the Istio itself but rather to the misconfiguration of the Istio custom resources (CRs) or other cluster resources.
+
+## Network connectivity
+
+1. Check if Istio CR is in Warning state, if yes check for the warning message and conditions. It might be helpful at the beginning of the investigation.
+2. Verify that no [NetworkPolicies](https://kubernetes.io/docs/concepts/services-networking/network-policies/) is affecting the connectivity by blocking traffic between pods in the mesh. You can find all network policies by running the command: `kubectl get networkpolicies -A`
+3. The configuration of resources of the following kinds can affect the connectivity in the service mesh. Verify that those resources are configured as intended:
+    - [`DestinationRule`](https://istio.io/latest/docs/reference/config/networking/destination-rule/)
+    - [`PeerAuthentication`](https://istio.io/latest/docs/reference/config/security/peer_authentication/)
+    - [`Gateway`](https://istio.io/latest/docs/reference/config/networking/gateway/)
+    - [`AuthorizationPolicy`](https://istio.io/latest/docs/reference/config/security/authorization-policy/)
+    - [`RequestAuthentication`](https://istio.io/latest/docs/reference/config/security/request_authentication/)
+4. Use `istioctl analyze -A` to check for potential issues in the Istio configuration. This command checks for potential issues in the Istio configuration and provides suggestions on how to fix them.
+5. Check response flags: DC (Downstream client terminated connection), UC (Upstream terminated connection) are out of scope for Istio team, since it relates to client(DC) or workload application(UC) behavior. You can find them in the Envoy sidecar's access logs.
```

Comment:
I think it would be good to have a link how to enable access logs
```suggestion
5. Enable the access logs of the Envoy Proxy by following this [guide](https://istio.io/latest/docs/tasks/observability/logs/access-log/). In the access log you can find the field `response_flags`. The response flags DC (Downstream client terminated connection) and UC (Upstream terminated connection) are out of scope for Istio team, since it relates to client(DC) or workload application(UC) behaviour.
```